README.md

```diff
@@ -1,2 +1,177 @@
+
+> The key features of this application:
+
+- **My informations**
+- **My works**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+[Click here for watching project presentation](https://www.loom.com/share/a7033791d19d43eb9ee28184478c0441)
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> You can have access to my project online by clicking on the below link:
+
+- [Live Demo Link](https://batoorsohail.github.io/microverse-course/index.html)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> If you want to use my project you should clone it into your computer and then open it in your code editor.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- GitHub 
+- Install Git 
+- Code Editor
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="Sohail Batoor"></a>
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/batoorsohail)
+- Twitter: [@twitterhandle](https://www.linkedin.com/in/sohail-batoor-52429b230/)
+- LinkedIn: [LinkedIn](https://twitter.com/sohailBatoor)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Program**
+- [ ] **Join**
+- [ ] **News**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+This project is the second project that I make. If you like my project please give it a Star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for helping me to complete this project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **Can we clone it without Git?**
+
+  - No, you should install Git.
+
+- **Which programs did you use to make this?**
+
+  - HTML, CSS and JavaScript
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```

Comment:
- [ ] Please remove this after you are done updating your README file as it is just a suggestion on how to add the LICENSE file. 🥸️

Reply:
Fixed.